src/processing/vFramer/include/vDraw.h

```diff
@@ -125,6 +131,34 @@ class accDraw : public vDraw {
 
 };
 
+class rasterDraw : public vDraw {
+
+protected:
+    #define XLIMIT 304          //array needs constants
```

Comment:
The raster drawer won't work with hard coded resolutions. We have to be flexible with resolutions